pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -2072,3 +2081,15 @@ func LegacySaveCommandToUnstructured(cmd *dashboards.SaveDashboardCommand, names
 
 	return finalObj, nil
 }
+
+func getFolderUIDs(hits []dashboardv0alpha1.DashboardHit) []string {
+	folderUIDs := []string{}
+	folderSet := map[string]bool{} // to avoid duplicates
```

Comment:
I don't think you need both, just keep the set and use `maps.Keys(folderSet)` if you need the ids.

Reply:
right but doesn't map.Keys loop again?  was trying to avoid another loop, especially with large search results

Reply:
I don't think performance is an issue here, loops are pretty fast in golang, see https://go.dev/play/p/Dlk1OwQGEGA